R/highlight_function.R

```diff
@@ -0,0 +1,48 @@
+#' Highlight one discrete group in a comparison graph
+#'
+#' Pick the function depending on the aesthetic of your ggplot object (fill or color).
+#' Specify the unique factor name of the group you're highlighting (as a case-sensitive string), and the vector it's found in.
+#'
+#' @param variable group vector, same as what is in your color/fill aesthetic. Include as 'data$variable' format
+#' @param highlight name of group of interest, as character string
+#'
+#' @examples
+#' ggplot(dplyr::filter(transit_ridership, year=="2019"),
+#'        aes(x = system, y = ridership, fill = system)) +
+#'    geom_col() +
+#'    cmap_fill_highlight(transit_ridership$system, "metra")
+#'
+#' @export
+cmap_fill_highlight <- function(variable, highlight) {
+
+  n <- length(unique(variable))
+
+  palette <- rep.int(c(other = "#9daab3"), n)
```

Comment:
It seems like this palette color (and the highlight color below) could be useful to have as a functional argument? Defaulting to the ones you've chosen, but with the option to change if desired?